airbyte-integrations/connectors/source-snowflake/source_snowflake/source.py

```diff
@@ -182,28 +186,30 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
             stream_class = TableChangeDataCaptureStream
             push_down_filters_class = PushDownFilterChangeDataCaptureStream
 
-        standard_streams = self._get_standard_streams(stream_class, config, url_base, table_catalog_stream, authenticator)
+        standard_streams = self._get_standard_streams(stream_class, config, url_base, table_catalog_stream, authenticator, time_zone_offset)
         push_down_filters_streams = self._get_push_down_filters_streams(push_down_filters_class,
-                                                                        config, url_base, authenticator, standard_streams)
+                                                                        config, url_base, authenticator, standard_streams, time_zone_offset)
         streams = list(standard_streams.values()) + push_down_filters_streams
 
         return streams
 
     @classmethod
     @handle_no_permissions_error
-    def _get_standard_streams(cls, stream_class, config, url_base, table_catalog_stream, authenticator):
+    def _get_standard_streams(cls, stream_class, config, url_base, table_catalog_stream, authenticator, time_zone_offset):
         standard_streams = {}
         for table_object in table_catalog_stream.read_records(sync_mode=SyncMode.full_refresh):
             standard_stream = stream_class(url_base=url_base,
                                            config=config,
                                            authenticator=authenticator,
-                                           table_object=table_object)
+                                           table_object=table_object,
+                                           time_zone_offset=time_zone_offset)
             standard_streams[standard_stream.name] = standard_stream
 
         return standard_streams
 
     @classmethod
-    def _get_push_down_filters_streams(cls, push_down_filter_stream_class, config, url_base, authenticator, standard_streams):
+    def _get_push_down_filters_streams(cls, push_down_filter_stream_class, config, url_base, authenticator, standard_streams,
+                                       time_zone_offset):
         push_down_filters_streams = []
         if 'streams' in config and config['streams']:
```

Comment:
It should be enough to have just:

```python
if config.get('streams'):
   ...
   # magic here
```

and better way is to have the variable instead, that we can reuse later on:
```python
config_streams = config.get('streams')
if config_streams:
   ...
   # magic here
    for push_down_filter_stream_config in config_streams:
        ...
        # another layer of magic
```

Reply:
Done